packages/tachyons/src/modules/_rangeSlider.css

```diff
@@ -10,12 +10,14 @@
 .rangeSlider__horizontal {
   height: 100%;
   width: 100%;
+  cursor: col-resize;
```

Comment:
You also need to set this on the actual handle, same with the vertical one. The handle has cursor: pointer atm which has higher specificity than this rule.